rest_registration/signals.py

```diff
@@ -9,3 +9,6 @@
 
 # A user has activated his or her account.
 user_activated = Signal(providing_args=["user", "request"])
+
+# A user has verified his or her new email.
+user_changed_email = Signal(providing_args=["user", "request"])
```

Comment:
I think it would be nice to provide the `old_email`, `new_email` as parameters as well. so something like `providing_args=["user", "new_email", "old_email", "request"]`.